src/js/tabs/firmware_flasher.js

```diff
@@ -291,6 +291,7 @@ firmware_flasher.initialize = async function (callback) {
                 "USE_SERIALRX_FPORT",
                 "USE_SERIALRX_GHST",
                 "USE_SERIALRX_JETIEXBUS",
+                "USE_SERIALRX_MAVLINK",
```

Comment:
If we do not add this here - we can enable telemetry - otherwise somehow telemetry cannot be enabled

```suggestion
```